server/src/main/java/org/eclipse/openvsx/adapter/VSCodeAdapter.java

```diff
@@ -157,6 +150,8 @@ public ExtensionQueryResult extensionQuery(@RequestBody ExtensionQueryParam para
                 throw new ResponseStatusException(HttpStatus.BAD_REQUEST, exc.getMessage(), exc);
             }
         }
+
+        extensions = excludeBuiltInExtensions(extensions);
         if(totalCount == null) {
             totalCount = (long) extensions.size();
```

Comment:
`totalCount` is incorrect as it's assigned in line 136 and this doesn't execute, is it safe to remove that line and always assign `totalCount` here?